packages/host/app/components/operator-mode/code-submode/playground/playground-panel.gts

```diff
@@ -191,11 +263,13 @@ export default class PlaygroundPanel extends Component<Signature> {
   }
 
   private get canEditCard() {
-    return Boolean(
-      this.format !== 'edit' &&
-        this.card?.id &&
-        this.realm.canWrite(this.card.id),
-    );
+    // in error case, we may not have `this.card?.id`, but we can get the id from `this.cardResource?.id`
+    let cardId = this.cardResource?.id;
+    return cardId && this.realm.canWrite(cardId);
```

Comment:
the realm service is not going to know how to deal with a local ID. however, the instance does have a realm URL meta field. a more reliable solution here would be to pass that realm url meta field value for the instance instead of assuming you are dealing with a remote id.